### ISSUE TITLE
"Enable selective registration for UI Automation events and property changes" requires a restart of NVDA to take effect, which needs to be reflected on the GUI.

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#13505]
Based on [this comment][1] in #13250.
[1]: https://github.com/nvaccess/nvda/issues/13250#issuecomment-1061658399

### Steps to reproduce:
Use the steps from #13250 to test, then turn off "selective registration for UI Automation events and property changes" in the advanced settings panel.

### Actual behavior:
This option actually requires a restart of NVDA to take effect.

### Expected behavior:
It is best to take effect in time, but if this is not possible, it needs to be reflected in the GUI.
#### NVDA installed/portable/running from source:
 installed, portable, running from source
#### NVDA version:
all
#### Windows version:
n/a
#### Name and version of other software in use when reproducing the issue:
n/a
#### Other information about your system:
n/a
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by leonardder** [comment]
While this is an advanced option and therefore I disagree that it should be reflected in the GUI, I agree that it makes sense to document this in the user guide.

**Comment by cary-rowen** [comment]
Hi, 
It wouldn't be very expensive to implement this in the GUI, and I think it might be a user experience improvement if it was reflected in the GUI.
Of course, there are changes to be made at the same time in the user guide.
Thanks

**Comment by CyrilleB79** [comment]
I also support to have this information in the GUI, as well as in the User Guide. If there is no other downside, having it in the GUI can only make things clearer.

For me the fact that the option is in the advanced setting panel should not be a reason to have a poorer documentation. If the checkbox is easily discoverable, its limitations should also be.

@leonardder if there is any other downside to add this information in the GUI, could you please elaborate? Thanks.

**Comment by leonardder** [comment]
There's another option that requires a restart "Enable loading custom 
code from Developer Scratchpad directory" where it isn't part of the gui.

**Comment by CyrilleB79** [comment]
> There's another option that requires a restart "Enable loading custom code from Developer Scratchpad directory" where it isn't part of the gui.

No, you can just reload the plugins with NVDA+ctrl+F3 and the new setting is already taken into account. Of course, one should know that plugins and add-ons are loaded at startup or when an explicit reload is requested (NVDA+ctrl+F3).
But this seemed quite obvious for me that plugins are only loaded at startup or upon NVDA+ctrl+F3.
On the contrary, knowing that registration for UI Automation events is only done at startup and cannot be changed afterwards is much less obvious for me.
But maybe it's just a matter of habit because I am used to working with plugins and add-ons, but not with UIA events?

**Comment by CyrilleB79** [comment]
While resolving this issue, double check if there is another option needing a restart to document it the same way. I cannot think to such an option however.
I have no knowledge about the following options that may be double-checked for this:
* Enable support for HID braille
* Diff algorithm
